exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/EmbeddedQueryQueue.java

```diff
@@ -93,7 +93,7 @@ public void release() {
     public String queueName() { return "local-queue"; }
   }
 
-  private final int queueTimeoutMs;
+  private final long queueTimeoutMs;
```

Comment:
It is not necessary for the timeout to be a long. Timeouts should be on the order of minutes, not decades, so an int is perfectly fine and represents the semantics just a bit better.

Reply:
Fixed